filestack/mixins/filestack_imagetransform_mixin.py

```diff
@@ -143,7 +143,11 @@ def av_convert(self, preset=None, force=None, title=None, extname=None, filename
         response = utils.make_call(transform_url, 'get')
 
         if response.ok:
-            return filestack.models.AudioVisual(transform_url, apikey=new_transform.apikey, security=new_transform.security)
+            uuid = response.json()['uuid']
```

Comment:
I'd change this to something like this:
```python
if not response.ok:
    raise Exception

return AudioVisual
```

Less indented lines + return at the end seems more intuitive